### PR TITLE
[WIP] Detector related changes

### PR DESCRIPTION
**Pull request by kichalla**

### CODE REVIEW

File: Oryx.sln [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.30011.22
+# Visual Studio 15
```
Comment (kichalla) [inline Oryx.sln:3]: weird not sure why this changed. will revert it

File: vsts/pipelines/templates/_signBinaryDetector.yml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    dotnet new globaljson --sdk-version $(DotNetCoreSdkVersion)
+
+    Write-Host "Setting up git_commit and build_number as env variable"
+    $env:GIT_COMMIT=$(git rev-parse HEAD)
+    $env:BUILD_NUMBER=$env:BUILD_BUILDNUMBER
+    $projectFile="src\Detector\Detector.csproj"
+    
+    dotnet build Detector.sln -c $(BuildConfiguration)
+  displayName: 'Build Detector.sln'
+  condition: and(succeeded(), eq(variables['setSignTypeVariable.SignType'], 'real'))
```
Comment (qianz2) [inline vsts/pipelines/templates/_signBinaryDetector.yml:55]: Thanks for adding the builds and signing for the detector, this part looks great!

File: build/buildSln.sh [hunk 3]
```diff
@@ -6,13 +6,15 @@
 
 set -e
 
+solutionFileName="$1"
+
```
Comment (qianz2) [inline build/buildSln.sh:12]: I see here we can make it dynamic to build, great

**Comment by kichalla** [comment]
Closing this PR and opening 2 new ones to minimize the number of changes to review

**Pull request closed without merging**